component/org.wso2.carbon.identity.oauth2.dpop/src/main/java/org/wso2/carbon/identity/oauth2/dpop/internal/DPoPServiceComponent.java

```diff
@@ -49,6 +54,15 @@ public class DPoPServiceComponent {
     @Activate
     protected void activate(ComponentContext context) {
 
+
+        //TODO: Remove true and replace with the actual table check
+        boolean isAvailableTable = true; //&& IdentityDatabaseUtil.isTableExists(DPOP_JKT_TABLE_NAME);
```

Comment:
let's fix this

Reply:
setting `isAvailableTable` to `false` for now.this will be replaced with the actual table check when the jdbc dependancy issue is fixed.